README.md

```diff
@@ -1,7 +1,25 @@
-Brief Description:
+# Boston Burger Reviews
+<hr>
+*A first stop for the best Boston-based burgers.*
```

Comment:
This is displaying the beginning and ending asterisks `*`, If you want the text italic I would replace the asterisks with underscores `_`